meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: python-dlinfo
+  version: 1.2.1
```

Comment:
Since the release of v1.2.1 the "master" branch underwent uesr-facing changes (see `CHANGELOG.md`).
"1.2.1" is thus no longer correct.
Can you inject the version string produced by `setuptools-scm` here?